oauth.ts

```diff
@@ -97,6 +97,39 @@ const WELL_KNOWN_OAUTH_APPS: WellKnownOAuthApps = {
     clientIdKey: "OAUTH_CLIENT_ID_GITHUB",
     clientSecretKey: "OAUTH_CLIENT_SECRET_GITHUB",
   },
+  "google": {
+    clientIdKey: "OAUTH_CLIENT_ID_GOOGLE",
+    clientSecretKey: "OAUTH_CLIENT_SECRET_GOOGLE",
+  },
+};
+
+const extractProviderFromAppName = (appName: string): string | null => {
```

Comment:
I think the app could implement a loader which returns a provider name. But for now, LGTM